.gitignore

```diff
@@ -2,3 +2,7 @@ deploy/local.properties
 src/private/*.js
 src/private/*.sjs
 .DS_Store
+data/*
```

Comment:
I don't think you want to ignore the data directory. It's useful, for development purposes, to have some sample data to work with.